<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchematicEditor</class>
 <widget class="QMainWindow" name="SchematicEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schematic Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CADView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Project"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionToolSelect"/>
    <addaction name="actionToolMove"/>
    <addaction name="actionToolDrawText"/>
    <addaction name="actionToolDrawRectangle"/>
    <addaction name="actionToolDrawPolygon"/>
    <addaction name="actionToolDrawCircle"/>
    <addaction name="actionToolDrawEllipse"/>
    <addaction name="separator"/>
    <addaction name="actionToolDrawWire"/>
    <addaction name="actionToolAddComponent"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClose_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="actionPrint"/>
   <addaction name="actionPDF_Export"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Control_Panel"/>
   <addaction name="actionShow_Board_Editor"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QToolBar" name="viewToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_All"/>
  </widget>
  <widget class="QToolBar" name="editToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionRotate_CCW"/>
   <addaction name="actionRotate_CW"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionRemove"/>
  </widget>
  <widget class="QToolBar" name="toolsToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolSelect"/>
   <addaction name="actionToolMove"/>
   <addaction name="separator"/>
   <addaction name="actionToolDrawText"/>
   <addaction name="actionToolDrawRectangle"/>
   <addaction name="actionToolDrawPolygon"/>
   <addaction name="actionToolDrawCircle"/>
   <addaction name="actionToolDrawEllipse"/>
   <addaction name="separator"/>
   <addaction name="actionToolDrawWire"/>
   <addaction name="actionToolAddComponent"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/quit.png</normaloff>:/img/actions/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/save.png</normaloff>:/img/actions/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/info.png</normaloff>:/img/actions/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/print.png</normaloff>:/img/actions/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPDF_Export">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/pdf.png</normaloff>:/img/actions/pdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF Export</string>
   </property>
  </action>
  <action name="actionShow_Control_Panel">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/home.png</normaloff>:/img/actions/home.png</iconset>
   </property>
   <property name="text">
    <string>Show Control Panel</string>
   </property>
  </action>
  <action name="actionShow_Board_Editor">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/chip.png</normaloff>:/img/actions/chip.png</iconset>
   </property>
   <property name="text">
    <string>Show Board Editor</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/undo.png</normaloff>:/img/actions/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/redo.png</normaloff>:/img/actions/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/zoom_in.png</normaloff>:/img/actions/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/zoom_out.png</normaloff>:/img/actions/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_All">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/zoom_all.png</normaloff>:/img/actions/zoom_all.png</iconset>
   </property>
   <property name="text">
    <string>Zoom All</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/help.png</normaloff>:/img/actions/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRotate_CCW">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/rotate_left.png</normaloff>:/img/actions/rotate_left.png</iconset>
   </property>
   <property name="text">
    <string>Rotate counterclockwise</string>
   </property>
  </action>
  <action name="actionRotate_CW">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/rotate_right.png</normaloff>:/img/actions/rotate_right.png</iconset>
   </property>
   <property name="text">
    <string>Rotate clockwise</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/copy.png</normaloff>:/img/actions/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/cut.png</normaloff>:/img/actions/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/paste.png</normaloff>:/img/actions/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/delete.png</normaloff>:/img/actions/delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/close.png</normaloff>:/img/actions/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionToolSelect">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/select.png</normaloff>:/img/actions/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionToolAddComponent">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/chip.png</normaloff>:/img/actions/chip.png</iconset>
   </property>
   <property name="text">
    <string>Add Component</string>
   </property>
  </action>
  <action name="actionToolDrawWire">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/drawWire.png</normaloff>:/img/actions/drawWire.png</iconset>
   </property>
   <property name="text">
    <string>Draw Wire</string>
   </property>
  </action>
  <action name="actionToolMove">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/move.png</normaloff>:/img/actions/move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionToolDrawText">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/draw_text.png</normaloff>:/img/actions/draw_text.png</iconset>
   </property>
   <property name="text">
    <string>Draw Text</string>
   </property>
   <property name="toolTip">
    <string>Draw Text</string>
   </property>
  </action>
  <action name="actionToolDrawPolygon">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/draw_polygon.png</normaloff>:/img/actions/draw_polygon.png</iconset>
   </property>
   <property name="text">
    <string>Draw Polygon</string>
   </property>
  </action>
  <action name="actionToolDrawRectangle">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/draw_rectangle.png</normaloff>:/img/actions/draw_rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Rectangle</string>
   </property>
  </action>
  <action name="actionToolDrawCircle">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/draw_circle.png</normaloff>:/img/actions/draw_circle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Circle</string>
   </property>
  </action>
  <action name="actionToolDrawEllipse">
   <property name="icon">
    <iconset resource="../../../img/images.qrc">
     <normaloff>:/img/actions/draw_ellipse.png</normaloff>:/img/actions/draw_ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Draw Ellipse</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CADView</class>
   <extends>QGraphicsView</extends>
   <header location="global">common/cadview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../img/images.qrc"/>
 </resources>
 <connections/>
</ui>
